roles/kubernetes-apps/ansible/templates/coredns-deployment.yml.j2

```diff
@@ -6,9 +6,7 @@ metadata:
   namespace: kube-system
   labels:
     k8s-app: coredns{{ coredns_ordinal_suffix | default('') }}
-    kubernetes.io/cluster-service: "true"
```

Comment:
why we want to remove it as cluster-service ?

Reply:
It's not defined in the templates from CoreDNS, but it is set on the service, and not the deployment it self